patches/chromium/feat_add_set_theme_source_to_allow_apps_to.patch

```diff
@@ -0,0 +1,91 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@slack-corp.com>
+Date: Mon, 26 Aug 2019 14:32:41 -0700
+Subject: feat: add set_theme_source to allow apps to
+ override chromiums internal theme choice
+
+
```

Comment:
This patch requires a justification. Please provide one or more of:

1. A reason this can't be done without a patch
2. Instructions for how to remove this patch by upstreaming or alternative implementation
3. How to know if this patch is still necessary